Source/Core/VideoCommon/CPMemory.cpp

```diff
@@ -33,3 +33,62 @@ void CopyPreprocessCPStateFromMain()
 {
   memcpy(&g_preprocess_cp_state, &g_main_cp_state, sizeof(CPState));
 }
+
+std::pair<std::string, std::string> GetCPRegInfo(u8 cmd, u32 value)
+{
+  // Most things here seem to contradict YAGCD; for example LoadCPReg seems to assume that there is
+  // only one vertex descriptor and anything starting with 0x50 or 0x60 writes it, while YAGCD says
+  // there are 8 of them.  Not sure what the deal with that is.
+  switch (cmd & 0xf0)
+  {
+  case 0x30:
```

Comment:
Can we turn these magic numbers into a variable?  I'm surprised they don't exist somewhere already?

Reply:
I guess the string is enough if these don't already exist in Dolphin, just surprised

Reply:
I've created constants for these and used them in all the contexts that I could find.  I've also moved my comments about inconsistencies with YAGCD to those constants.